meinberlin/apps/users/templates/meinberlin_users/user_indicator.html

```diff
@@ -1,61 +1,55 @@
-                <li>
-                    <a class="button button--light" href="{% url 'account_login' %}?next={{ redirect_field_value|urlencode }}" role="menuitem">{% translate "Login" %}</a>
-                </li>
-                <li>
-                    <a href="{% url 'account_signup' %}?next={{ redirect_field_value|urlencode }}" role="menuitem">{% translate "Register" %}</a>
-                </li>
-                <li>
-                    <a href="{% url 'account_reset_password' %}?next={{ redirect_field_value|urlencode }}" role="menuitem">{% translate "Forgot password?" %}</a>
-                </li>
-            </ul>
-        </div>
-    {% endif %}
+            <button data-dropdown-trigger type="button" aria-haspopup="menu" aria-controls="dropdown-menu" tabindex="0">
+                <i class="fas fa-user text--color-primary" aria-hidden="true"></i>
+                {{ request.user.username }}
+            </button>
+            <div role="menu" class="dropdown__menu" aria-label="Account" tabindex="-1" data-dropdown-menu>
+                <h2>{% translate "My account" %}</h2>
+                <ul class="list--clean">
+                    <li class="dropdown__item">
+                        <a href="{% url 'account' %}" role="menuitem">{% translate "Account Settings" %}</a>
+                    </li>
+                    {% for organisation in request.user.organisations %}
+                        <li class="dropdown__item">
+                            <a href="{% url 'a4dashboard:project-list' organisation_slug=organisation.slug %}" role="menuitem">{{ organisation.name }}</a>
+                        </li>
+                    {% endfor %}
+                    {% if request.user.is_superuser %}
+                        <li class="dropdown__item">
+                            <a href="{% url 'meinberlin_platformemails:create' %}" role="menuitem">{% translate "Platform Email" %}</a>
+                        </li>
+                    {% endif %}
+                    <li class="dropdown__item">
+                        <form class="block--nogap text--color-dark" action="{% url 'account_logout' %}" method="post" aria-label="{% translate 'Logout' %}" role="menuitem">
+                            {% csrf_token %}
+                            <input type="hidden" name="next" value="{{ redirect_field_value }}">
+                            <button class="button button--light" type="submit">{% translate "Logout" %}</button>
+                        </form>
+                    </li>
+                </ul>
+            </div>
+        {% else %}
+            <button data-dropdown-trigger type="button" aria-haspopup="menu" aria-controls="dropdown-menu" tabindex="0">
+                <i class="fas fa-user text--color-primary" aria-hidden="true"></i>
+                {% translate "Login" %}
+            </button>
+            <div role="menu" class="dropdown__menu" aria-label="Login of Register" tabindex="-1" data-dropdown-menu>
```

Comment:
Nothing new but should this be "Login **or** Register" ? And it should also be translated, right?